### DISCUSSION TITLE
Screen off/on hotkey

### DISCUSSION THREAD

**Discussion by Wundrwaffle** [seerge/g-helper#3282] [Ideas]
Ok, ill try to explain what im looking for.

Is it possible function that will switch off lid without hibernating/sleep/waiting mode on? 
before Asus i was working with MSI, and there was not problem to close lid or via Fn-key and it keep running sound/video to external displays, but now with new laptop, its impossible (tried to tweak settings with battery, power management, etc)
When i attach external displays to laptop and close lid, external display also switch off.
And yeah, no Fn + button that switch off display on board (((

Asus Scar 17 733PYV

**Comment by seerge** [comment]
@Wundrwaffle 

What you talk about is just setting in windows control panel. Sed Lid Close action to `Do nothing`
![Screenshot 2024-10-22 112730](https://github.com/user-attachments/assets/8983a97a-1a3a-4a33-8cdc-ca1e2d216535)

Furthermore G-Helper has a `Clamshell mode` (in `Extra`) that would toggle this setting ON automatically when you are plugged in + have external display connected and OFF when you unplug or disconnect monitor.

**Reply by Wundrwaffle** [reply]
Oh. Got it, will check extras.

In my case, i tried different standart settings, and even there option "do nothing" it still gets into kind of sleep mode, display is off but also plugged devices that show main screen is off, and when open lid there welcome screen with user choose menu

**Reply by Valour549** [reply]
Having upgraded to a new laptop recently, I have ran into the same issue as you. The difference is I don't use external screen, I just want my laptop to not go into pseudo-sleep whenever the laptop screen is off, which it keeps doing. In your case, Clamshell mode may indeed just solve your problem, if not, keep reading.

Basically the root cause of this is _Modern Standby_. With MS on, after your screen has been off for a lil while, it will ALWAYS go into this pseudo-sleep mode. It doesn't matter how your screen turns off: it could be due to inactivity, it could be you used a button to turn it off, it could be because you closed the lid. When your screen turns off initially, you could move the mouse and the screen comes back on. But after an unspecified amount of time, this pseudo-sleep caused by MS will activate and when you move the mouse you'll have to log in to your account again. I don't like this—if I want to lock my PC I can lock it manually, if I want it to sleep I can set a timer or do it manually, same with hibernation.

Here is a definitive list to solving this problem, you can try one by one to see if it solves your problem, but I prefer doing everything:

• Disable [**Automatically Lock Computer after Inactivity in Windows 11**](https://www.elevenforum.com/t/automatically-lock-computer-after-inactivity-in-windows-11.4774/)

• Disable **When my PC is inactive, show the lock screen instead of turning off the screen**
1) Go to regedit `Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Lock Screen`
2) Create a new D-word called `SlideshowAutoLock` and give it the value `0`.
3) This is the same thing as going to Windows Settings → Personalization → Lock Screen → Personalize your lock screen. Now select "Slideshow" and in the Advanced slideshow settings, untick _When my PC is inactive, show the lock screen instead of turning off the screen_. After doing this, you can change Personalize your lock screen back to "Windows spotlight" or "Picture". Yeah, crazily hidden I know.

• Disable [**Modern Standby**](https://www.elevenforum.com/t/disable-modern-standby-in-windows-10-and-windows-11.3929/)

• Disable the [**Lock Screen**](https://www.elevenforum.com/t/enable-or-disable-lock-screen-in-windows-11.1287/). This one is optional. You will still be able to lock your PC, the only difference is when you wake it up there is no lock-screen and you will be immediately presented with the log-on screen.

**Reply by IlyaDin** [reply]
> @Wundrwaffle
> 
> What you talk about is just setting in windows control panel. Sed Lid Close action to `Do nothing` ![Screenshot 2024-10-22 112730](https://private-user-images.githubusercontent.com/5920850/378775608-8983a97a-1a3a-4a33-8cdc-ca1e2d216535.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI0NDA5MTAsIm5iZiI6MTczMjQ0MDYxMCwicGF0aCI6Ii81OTIwODUwLzM3ODc3NTYwOC04OTgzYTk3YS0xYTNhLTRhMzMtOGNkYy1jYTFlMmQyMTY1MzUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjRUMDkzMDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MmZlNTI5MmE4YTM0NzMwZjU2NThlZDRhYTJjOTk2MGVmYzVjZjUyYjE3MmMzMmNlMTg1MDI3NTc0OWJkZmMxZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.2Y9X1XmE7tQhWZEsa2Tib8ZDfnRc_stDhKT3c_IisTI)
> 
> Furthermore G-Helper has a (in ) that would toggle this setting ON automatically when you are plugged in + have external display connected and OFF when you unplug or disconnect monitor.`Clamshell mode``Extra`

Hello, Serge! I also face the problem of turning off the laptop screen. I don't want the screen to be on during rendering or other background work. On my old Asus, I could press Fn+F7 and the screen would turn off, but the processes would continue to run. There is no such button on the new ROG 18, and the Toggle Screen function doesn't work as expected. Maybe I didn't understand how to configure it properly? In all modes, the "do nothing" option is selected.

**Reply by seerge** [reply]
@IlyaDin hello, the only known way to turn screen off is also locks the screen in windows (this is what you can assign to any M-key via Extra settings). But that will likely stop some processes from running.

It could be that your old device had some proprietary way you toggle screen from Asus themselves, but if it doesn’t work same on new model, most likely it’s not supported by it.

**Reply by IlyaDin** [reply]
> здравствуйте, единственный известный способ выключить экран это также блокировка экрана в windows (это то, что вы можете назначить на любой M-key через Дополнительные настройки). Но это, скорее всего, остановит выполнение некоторых процессов.
> 
> Возможно, на вашем старом устройстве был какой-то фирменный способ переключения экрана от Asus, но если он не работает так же на новой модели, скорее всего, он им не поддерживается.

Thanks for all your hard work!
